elements/campaign-product-shot/campaign-product-shot.js

```diff
@@ -0,0 +1,50 @@
+import {
+  registerElement,
+  BulbsHTMLElement,
+} from 'bulbs-elements/register';
+import { filterBadResponse, getResponseJSON } from 'bulbs-elements/util';
+import invariant from 'invariant';
+
+class CampaignProductShot extends BulbsHTMLElement {
+
+  handleRequestSuccess (data) {
+    this.innerHTML =
+      `<div class='campaign-product-shot'>
+        <img src='${data.product_shot_url}'>
```

Comment:
Yeah, you could just use invariant to enforce it: `invariant(data.product_shot_url, 'CampaignProductShot.handleRequestSuccess(data): data.product_shot_url is undefined')`